.github/workflows/wheel_win_x86_64.yml

```
@@ -160,6 +151,36 @@ jobs:
           }
           cd ..
 
+      - name: Determine Python version
+        id: pyvs
+        shell: bash
+        run: |
+          echo "version=$(echo  ${{ matrix.cibw_build }} | tr -cd '[:digit:].' | sed 's/./&./1')" >> $GITHUB_OUTPUT
+
+      - uses: actions/setup-python@v5
+        name: Install Python
+        with:
+          python-version: ${{ steps.pyvs.outputs.version }}
+
+      - name: Test wheels
+        run: |
+          python -m ensurepip --upgrade
+          python -m pip install setuptools
+          python -m pip install -r requirements-tests.txt
+          if ('${{ matrix.pl_backend }}' -eq 'lightning_kokkos'){
+            $SKIP_COMPILATION=true
+            $PL_BACKEND="lightning_qubit"
+            python -m pip install -e . -vv
```

I think we need to either `export` the variables or set them on a single line
```suggestion
            SKIP_COMPILATION=true \
            PL_BACKEND="lightning_qubit" \
            python -m pip install -e . -vv
```